### ISSUE TITLE
libopencl_clang.so don't exist in open source build

### ISSUE THREAD

**Issue by dvrogozh** [intel/intel-graphics-compiler#73]
https://github.com/intel/intel-graphics-compiler/blob/4be564fa181ace300379a045db1fa6ea972e356e/IGC/BiFModule/CMakeLists.txt#L140

The library which we can build in open source from https://github.com/intel/opencl-clang in ocl-open-80 branch is named libcommon_clang.so while latest IGC master checks for libopencl_clang.so which eventually is missing if you build against system libraries.

This looks to be regression after d4f478be861e28b186992b2f0e40617f4fbf3da6 commit:
```
commit d4f478be861e28b186992b2f0e40617f4fbf3da6
Author: Wesierski, Lukasz <lukasz.wesierski@intel.com>
Date:   Fri Mar 1 23:27:33 2019 -0800

    Correct library name of common_clang

    Change-Id: Ied827e1bff0e564922a6bc5ea92406d689778b89
```

**Comment by dvrogozh** [comment]
This was fixed by a5a6528f247971ca19bca8fff2bf2dcdc7f3e710.

@lwesiers : thank you for the fix! You can try to use a bit of github automation next time: if you add the following to commit message, issue will be closed automatically:
`Fixes #73`

**Comment by iwwu** [comment]
available on igc_release_2019-03-12

**Comment by tjaalton** [comment]
this needs to be changed in IGC/CMakeLists.txt too, I'm not able to build IGC without setting -DCOMMON_CLANG_LIBRARY_NAME=common_clang

**Comment by dvrogozh** [comment]
Hm. Strange, this should have failed my build as well, but it did not. Not sure why. But yes, from the code this looks not fully fixed.

**Comment by dvrogozh** [comment]
I was cleaning everything, update and rebuild circle and reproduce this behavior. Indeed we need an update to IGC/CMakeLists.txt with s/opencl_clang/common_clang. @iwwu : can you, please, help to nail this down?

**Comment by iwwu** [comment]
@dvrogozh Sure, I will update when the change is merged

**Comment by iwwu** [comment]
Fixed on f61ab6d
